Makefile.docker

```diff
@@ -0,0 +1,82 @@
+.PHONY: all
+all:
+	@echo all
+
+# create tracee-make
+
+UID = $(shell id -u)
+GID = $(shell id -g)
+PWD = $(shell pwd)
+
+ALPINE_MAKE_CONTNAME = alpine-tracee-make
+ALPINE_MAKE_DOCKERFILE = docker/Dockerfile.alpine-tracee-make
+
+.PHONY: alpine-prepare
+alpine-prepare:
+	docker build \
+		-f $(ALPINE_MAKE_DOCKERFILE) \
+		-t $(ALPINE_MAKE_CONTNAME):latest \
+		--build-arg UID=$(UID) \
+		--build-arg GID=$(GID) \
+		.
+
+UBUNTU_MAKE_CONTNAME = ubuntu-tracee-make
+UBUNTU_MAKE_DOCKERFILE = docker/Dockerfile.alpine-tracee-make
+
+.PHONY: ubuntu-prepare
+ubuntu-prepare:
+	docker build \
+		-f $(UBUNTU_MAKE_DOCKERFILE) \
+		-t $(UBUNTU_MAKE_CONTNAME):latest \
+		--build-arg UID=$(UID) \
+		--build-arg GID=$(GID) \
+		.
+
+# tracee-make's shell
+
+ifeq ($(STATIC), 1)
+    DOCKER_RUN_ENV = -e STATIC=1
+endif
+
+DOCKER_RUN_ARGS = run --rm --pid=host --privileged \
+		-v /etc/os-release:/etc/os-release-host:ro \
+		-v $(PWD):/tracee \
+		-v /lib/modules:/lib/modules:ro \
+		-v /usr/src:/usr/src:ro \
+		-v /tmp/tracee:/tmp/tracee \
+		-e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
+		$(DOCKER_RUN_ENV)
+
+.PHONY: alpine-shell
+alpine-shell:
+	docker $(DOCKER_RUN_ARGS) \
+		-it $(ALPINE_MAKE_CONTNAME) \
+		/bin/bash
```

Comment:
I think `/bin/bash` is not available in an alpine container (but `/bin/sh`)

Reply:
I made bash exist and be the default for the builder shell CMD only (with a profile to make it a better experience). The default for docker and other targets is and will still be sh only.